met-web/src/components/survey/building/index.tsx

```diff
@@ -138,19 +138,22 @@ const SurveyFormBuilder = () => {
                     {!isNameFocused ? (
                         <>
                             <MetHeader3
+                                variant="h3"
```

Comment:
Is the variant necessary?

Reply:
Marie Anik mentioned she wanted it to be a h3 tag so without the variant in the html code its a p tag

Reply:
I'm confused here, I though MetHeader3 is an h3 tag according to BC standards. Why would MetHeader3 be a p tag without the variant. Also we created Atomic headers like that to move away from the variant prop no?

Reply:
The problem was the material ui typography component is defaulted as p tag. If no variant is specified thats how it show up in the html code. No problem I will change the way the Met Headers are implemented in this pr to fix it.

Reply:
It seems this might take a bit more work then expected so Ill remove the variant and fix the header tags in a separate PR.